modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue

```diff
@@ -16,7 +16,7 @@
             </div>
 
             <div class="apos-login__body" v-show="loaded">
-              <form>
+              <form @submit.prevent="submit">
```

Comment:
is there affordance for a non-vue form submission here? otherwise not sure if we have to go through all the form fakery, unless we're doing that everywhere as well, in which case ignore

Reply:
@stuartromanek there is no affordance for a non-vue form submission. Not sure what you mean about the "form fakery," are we saying we don't like the form tag if we're not doing plain vanilla non-JS form submisisons? We wouldn't use it just for the semantics and allowing HTML5 form elements to know a form submission is happening and so on?